From 80fb6ada222a26e2f001fe682b5a933c109c1613 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <gqjiang@suse.com>
Date: Thu, 28 Jul 2016 14:03:51 +0800
Subject: [PATCH 8/8] md-cluster: remove EXPERIMENTAL info
Patch-mainline: Submitted to linux-raid http://www.spinics.net/lists/raid/msg53094.html
References: fate#316335

We can remove "EXPERIMENTAL" for md-cluster after
several rounds of development and bug fixes.

Signed-off-by: Guoqing Jiang <gqjiang@suse.com>
Acked-by: Guoqing Jiang <gqjiang@suse.com>
---
 drivers/md/Kconfig      | 2 +-
 drivers/md/md-cluster.c | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/md/Kconfig b/drivers/md/Kconfig
index 02a5345..74bc3ca 100644
--- a/drivers/md/Kconfig
+++ b/drivers/md/Kconfig
@@ -178,7 +178,7 @@ config MD_FAULTY
 
 
 config MD_CLUSTER
-	tristate "Cluster Support for MD (EXPERIMENTAL)"
+	tristate "Cluster Support for MD"
 	depends on BLK_DEV_MD
 	depends on DLM
 	default n
diff --git a/drivers/md/md-cluster.c b/drivers/md/md-cluster.c
index 361a69e..d0e605e 100644
--- a/drivers/md/md-cluster.c
+++ b/drivers/md/md-cluster.c
@@ -1266,7 +1266,6 @@ static struct md_cluster_operations cluster_ops = {
 
 static int __init cluster_init(void)
 {
-	pr_warn("md-cluster: EXPERIMENTAL. Use with caution\n");
 	pr_info("Registering Cluster MD functions\n");
 	register_md_cluster_operations(&cluster_ops, THIS_MODULE);
 	return 0;
-- 
2.6.2

